apps/server/src/config.ts

```diff
@@ -14,7 +14,9 @@ const nonSensitiveSharedConfig = {
   ServiceName: 'universal_widget',
   CryptoAlgorithm: 'aes-256-cbc',
   SophtronApiServiceEndpoint: 'https://api.sophtron.com/api',
-  SophtronVCServiceEndpoint: 'https://vc.sophtron.com/api/'
+  SophtronVCServiceEndpoint: 'https://vc.sophtron.com/api/',
+  InstitutionCacheListUrl: 'http://localhost:8088/institutions/cacheList',
```

Comment:
This will have to differ depending on whether it's local, staging, or production, so it should come from env